mods/ra2/rules/allied-infantry.yaml

```diff
@@ -162,6 +166,14 @@ e1:
 		Name: deployed-elite
 		Weapon: paraE
 		RequiresCondition: deployed && rank-elite
+    Armament@garrisoned:
```

Comment:
bad indention

Reply:
Which character are you using ? I tabulated all around the place...

Reply:
These are spaces. Use tabs (red on GitHub) everywhere.